src/models/dataElementsSet.ts

```diff
@@ -18,7 +18,7 @@ import { ValidationError } from "../utils/validations";
     # [... Array of data elements ...]
 */
 
-export const indicatorTypes = ["global", "sub", "custom"] as const;
+export const indicatorTypes = ["global", "sub", "custom", "reportable sub"] as const;
```

Comment:
As we discussed, we don't have a "reportableSub" indicator type, we keep the 3 we have. If we get a D2 dataElement which belongs to the group "Reportable Sub", then  we set indicatorType ="sub". 

So you have to tweak `const indicatorType = getGroupKey(groupCodes, degCodes, indicatorTypes)`. Currently is simply using a 1:1 map, now we have this special case.